src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -623,6 +626,24 @@ def get_worker_id(self) -> str:
         LOG.info(f"Obtained Worker ID: {worker_id}")
         return worker_id
 
+    def get_windows_user_secret_cmd(self, secret_id: str) -> str:
```

Comment:
nit: can we add a leading underscore to indicate that this is a private method of the class?

```suggestion
    def _get_windows_user_secret_cmd(self, secret_id: str) -> str:
```

Reply:
I was intending on reusing this function as part of my test for verifying the credentials. An example of this in use exists in one of my branches [here](https://github.com/hindleym/deadline-cloud-worker-agent/blob/hindleym-agent-user-password/test/e2e/test_installer.py#L187)

Reply:
Ah, that makes more sense. We can keep it then.